static/app-interpage.js

```
@@ -833,13 +833,82 @@
                     case 'memory_edited':
                         await handleMemoryEdited(event.data.catgirl_name);
                         break;
+                    case 'avatar_updated':
+                        // 从 Pet 窗口接收头像数据，注入到 Chat 窗口
+                        // 校验 lanlan_name：多角色场景下避免串头像
+                        if (event.data.lanlan_name && window.lanlan_config &&
+                            event.data.lanlan_name !== window.lanlan_config.lanlan_name) break;
+                        var incomingDataUrl = event.data.dataUrl || '';
```

Biome (2.4.10)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)